server/api/room/room.model.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var mongoose = require('mongoose'),
+    Schema = mongoose.Schema;
+
+var RoomSchema = new Schema({
+    name: String,
+    loc: String,                        //location
+    capacity: int,
```

Comment:
`int` isn't valid - use `Number`

Also just drop `location` - `name` is sufficient